microcosm_flask/swagger/decorators.py

```diff
@@ -0,0 +1,16 @@
+from microcosm_flask.swagger.parameters.decorated import SWAGGER_FORMAT, SWAGGER_TYPE
+
+
+def swagger_field(field, swagger_type="string", swagger_format=None):
```

Comment:
I found one usage of this "decorator" in our code. It's also not a proper decorator, which is something I'd like to fix. This comes at some expense to backwards compatibility, but I think it's justified.

Reply:
If it's in [argus](https://github.com/globality-corp/argus/blob/70ae4f97a5881fd0ede70e826f5608e2b28d77ab/argus/resources/troubleshooting_resources.py#L12), then we could probably drop it.

Reply:
I removed the `argus` usage, which leaves me free to make this work the way it should. Nice.